code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -208,6 +215,11 @@
 	message_admins("PA Control Computer turned [active ?"ON":"OFF"] by [usr ? key_name_admin(usr) : "outside forces"](<A HREF='?_src_=holder;adminmoreinfo=\ref[usr]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[usr]'>FLW</A>) in ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)",0,1)
 	log_game("PA Control Computer turned [active ?"ON":"OFF"] by [usr ? "[key_name(usr)]" : "outside forces"] in ([x],[y],[z])")
 	if(active)
+		radio.frequency = 1357
+		for (var/obj/machinery/field/containment/C in range(30, src))
+			fieldcount += C		
+		if (fieldcount.len<24)

```

Comment:
if(LAZYLEN(fieldcount) < 24)
Also 24 is a magic number that may not be accurate to the engine setup; you can do a tesla engine in a 3x3 box, and that's just 4 fields. You need a more robust check.